.circleci/config.yml

```
@@ -0,0 +1,68 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/python:3.5
+        environment:
+            PIPENV_VENV_IN_PROJECT: true
+    working_directory: ~/rep
```

Could you name it `repo`? It's already very short, but actually has a meaning.

```suggestion
    working_directory: ~/repo
```